hexa/databases/api.py

```diff
@@ -24,7 +24,7 @@ def format_db_name(name: str):
 
     # Ensure db_name do not start with a number
     if db_name[0] in "0123456789":
-        db_name.insert(0, "_")
+        db_name = db_name.replace(db_name[0], "_", 1)
```

Comment:
Doing it like this is over complicated and a bit strange. We do not want to replace a character, we want to add one to be sure to have the right format for PGSQL.
```python
db_name = "_" + db_name
```
It's easier to understand.